.github/workflows/R-CMD-as-cran-check.yaml

```diff
@@ -36,9 +36,18 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: any::rcmdcheck
-          needs: check
+          dependencies: NA
+          extra-packages: |
+            rcmdcheck
+            stan-dev/cmdstanr
+            testthat
+
+      - name: Install cmdstan
```

Comment:
I've just written a cached version of this for epinowcast (in github.com/epinowcast/actions) that we can either use directly or pull into the repo in order to slightly speed up CI